workloads/web.hbs.md

```diff
@@ -57,3 +57,38 @@ tanzu apps workload apply tanzu-java-web-app --type=web
 
 You can use this to test which applications can function well as serverless web applications,
 and which are more suited to the `server` application style.
+
+## <a id="communication"></a> Communication between `web` workloads
```

Comment:
I think this might better be called "Calling `web` workloads within a cluster" -- this applies even if a Deployment from a `worker` workload wants to call a `web` workload.